examples/jquery/bower.json

```diff
@@ -1,7 +1,8 @@
 {
   "name": "todomvc-jquery",
   "dependencies": {
-    "todomvc-common": "~0.3.0",
+    "todomvc-common": "~1.0.1",
+    "todomvc-app-css": "~1.0.1",
     "jquery": "~2.1.1",
     "handlebars": "~2.0.0",
     "director": "~1.2.2"
```

Comment:
Can you also add a .gitignore as seen here: https://github.com/tastejs/todomvc/pull/1111/files#diff-c39d5a21f6bb2f461f2549116f937d95R1